packages/nimble-components/src/theme-provider/tests/color.stories.ts

```diff
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/html';
 import { html, repeat } from '@microsoft/fast-element';
-import nimbleColorsMapJson from '@ni/nimble-tokens/dist/styledictionary/properties/colors.json';
+import nimbleColorsMapJson from '@ni/nimble-tokens/source/styledictionary/properties/colors.json';
```

Comment:
The colors story shouldn't be relying on module files that are not exported (i.e. shouldn't have knowledge of repo-specific behavior across packages).

Some options:
1. We do export the json
2. We export additional js files with more data, maybe exporting a JS object that looks like the JSON nested format https://amzn.github.io/style-dictionary/#/formats?id=jsonnested
3. Keep the existing JS export and do something like `import * as tokens  from '@ni/nimble-tokens/dist/styledictionary/js/tokens'` and `tokens.filter(tokens => tokens.startsWith('W'))` or something
4. Explicitly comment here that this is an internal import and create a tech debt issue. Or add a link to the following issue and a comment to the issue which is about adding package export fields (and break this current import behavior of non-exported files): https://github.com/ni/nimble/issues/172

Reply:
Think I kinda lean to 3 or 2, but 4 would be fine to kick the can down the road.